.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: "Main"
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+    tags-ignore:
+      - '**'
```

Comment:
Use this main file https://github.com/surfstudio/flutter-surf-logger/blob/main/.github/workflows/main.yml